fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/ToDate.java

```diff
@@ -70,4 +72,19 @@ public List<FunctionSignature> getSignatures() {
     public <R, C> R accept(ExpressionVisitor<R, C> visitor, C context) {
         return visitor.visitToDate(this, context);
     }
+
+    @Override
+    public boolean isPositive() {
+        return true;
+    }
+
+    @Override
+    public int getMonotonicFunctionChildIndex() {
+        return 0;
+    }
+
+    @Override
+    public Expression withConstantArgs(Expression literal) {
+        return new Date(literal);
```

Comment:
why don't return new ToDate(literal)? maybe adding Monotonic of Datev2, ToDate, ToDateV2 need some tests in partition prune. Can we choose to not add Monotonic for Datev2, ToDate, ToDateV2 in this pr?